### PR TITLE
Issue #13213: Remove '//ok' comments from methodlength

### PR DESCRIPTION
**Pull request by Acesif**
Issue #13213

Removed "//ok" comments from 
"InputMethodLengthCountEmptyIsFalse.java" and "InputMethodLengthCountEmptySmallSize.java" files

and also removed respective suppressions

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/InputMethodLengthCountEmptySmallSize.java [hunk 1]
```diff
@@ -10,11 +10,11 @@
 
 public class InputMethodLengthCountEmptySmallSize {}
 class AA {
-    AA() { // ok, 2 lines
+    AA() {
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/InputMethodLengthCountEmptySmallSize.java:13]: We are only requesting to remove "// ok" and not "// ok, _ _ _"

**Review by MANISH-K-07 — COMMENTED** [review]
@Acesif , You will have to remove the respective suppressions from checkstyle-input-suppressions.xml , Please see how merged PRs were done.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/InputMethodLengthCountEmptySmallSize.java (hunk 1 above)
Comment (Acesif) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/InputMethodLengthCountEmptySmallSize.java]: Thanks for pointing that out, I've made the appropriate changes

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/InputMethodLengthCountEmptySmallSize.java [hunk 2]
```diff
@@ -25,18 +25,18 @@ class AA {
...
+    void twoLines2() {
         oneLine();}
 
-    void twoLinesAndComment() { // ok
+    void twoLinesAndComment() {
         // some comment
     }
 
-    void oneLine() { oneLine();/* ok */ }
+    void oneLine() { oneLine();}
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/InputMethodLengthCountEmptySmallSize.java:39]: Please revert this

Comment (romani) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/InputMethodLengthCountEmptySmallSize.java:39]: Better to keep it , or change "ok" to some other word.

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**